stable/dynamic-gateway-service/templates/datapower-monitor.yaml

```diff
@@ -107,6 +107,10 @@ spec:
         resources:
 {{ toYaml .Values.datapowerMonitor.resources | indent 10 }}
 {{- if .Values.datapower.adminUserSecret }}
+        securityContext:
+          capabilities:
+            drop:
+            - ALL
```

Comment:
What's this for?

Reply:
It's for ICP linting requirements. I will move it to a different PR.